actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -829,6 +829,39 @@ def utf8_enforcer_tag
         '<input name="utf8" type="hidden" value="&#x2713;" />'.html_safe
       end
 
+      # Creates a text input linked to a datalist. A datalist contains a set of option tags.
+      # The datalist's bahavior is similar to a select tag. Datalists allow a selection from predefined values
+      # but are not constraint to those values. The text input can chosen from the datalist options or entered normally.
```

Comment:
Hey @ alexcameron89. Thanks for the feedback. I must have missed those when I was reading through it. 

I made a new commit to the PR with your changes.